src/test/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -88,4 +104,87 @@ public void propertyValidation() {
         }
     }
 
+    @Test
+    public void testKinesisClientLibConfigurationShouldGetProxyInfoFromPropertiesFile() throws Exception {
+        String PROPERTIES =
```

Comment:
Static variables are normally uppercase. Could you change this PROPERTIES to lower case properties.

Reply:
Agreed.  Just copied and pasted from the constructorTest()